cli/flox/src/commands/mod.rs

```diff
@@ -466,7 +477,7 @@ impl UpdateNotification {
             )
             .await
         } else {
-            Ok(None)
+            Ok(UpdateCheckResult::Skipped)
```

Comment:
nonblocking: should we skip writing the file in this case?

Reply:
I mean we _are_ skipping?
do you mean we shouldn't skip?

Reply:
Nope nvm confused myself